docs/plans/control-plane-migration-coverage-plan.md

```
@@ -0,0 +1,258 @@
+### Step 6 — Run the full suite
+
+```
+make test
+```
+
+This runs `manifests generate fmt vet setup-envtest` then the whole unit+
+integration suite (envtest binaries are fetched automatically; e2e excluded).
+Expect PASS. If envtest binary download fails in this environment (no network),
+fall back to a non-e2e package run:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)